unity-renderer/Assets/DCLPlugins/BuilderInWorld/Scripts/BuilderInWorldAudioHandler.cs

```diff
@@ -228,7 +228,15 @@ private void OnEntityBoundsCheckerStatusChanged(DCL.Models.IDCLEntity entity, bo
 
     private bool EntityHasBeenAddedSinceLastUpdate() { return (entityHandler.GetCurrentSceneEntityCount() > entityCount); }
 
-    private void OnDestroy() { Dispose(); }
+    private void OnDestroy()
+    {
+#if UNITY_STANDALONE || UNITY_EDITOR
+        if (DataStore.i.common.isWorldBeingDestroyed.Get())
```

Comment:
I'd rename this to `isApplicationQuitting`. 

`isWorldBeingDestroyed` can mean too many things.

Reply:
Agree! `isWorldBeingDestroyed `sound like `IsInMordor`, it could mean too many things xD

Reply:
Another proposal could be something like 

`isApplicationOnShutdown`